docs/Earn/liquid-staking/ksm/ksm-liquid-staking-mechanics.md

```diff
@@ -0,0 +1,187 @@
+
+* aKSMb ratio changes, thus all aKSMb balances “automatically” grow.
+
+Staking is well-described by its requirement, fees, rewards, validators, as well as smart contracts and workflow that shows the API endpoints to call to interact with these smart contracts.
+
+## Requirements
+
+The minimum requirements when staking are:
+
+* Minimum value to stake — 1 KSM.
+
+* Maximum value to stake — unlimited, at the user’s discretion.
+
+Staking can be repeated anytime, without waiting for the current staking to completed as staking on Polkadot is done by the ANKR-owned wallet where users send their stake directly.
+
+The minimum requirements when unstaking are:
+
+* Minimum value to unstake — 0 aKSMb.
+
+* Maximum value to unstake — staked amount + % of the reward, up to the user’s balance.
+
+* Release period — up to 28 days.
+
+## Fees
+
+When staking, the user pays:
+
+* 2% commission on the reward, to ANKR.
+
+When unstaking, the user pays no fees.
+
+The users must also count in the cost for outgoing transactions.
+
+## Rewards
+
+Validators/delegators receive rewards daily.
+
+ANKR backend calculates the ratio of rewards to the total stake amount weekly and publishes it to the aKSMb smart contract respectively.
+
+## Validators
+
+ANKR delegates the staked funds to trusted validators from Kusama, which may change in 1 day or longer.
+
+## Smart contract
+
+Smart contracts addresses involved in KSM staking are:
+
+Kusama deposit address:
+
+[ANKR deposit address](https://kusama.subscan.io/account/GUDy5gwmUTaHxcX8v2U3wyYdjSF8Zswnt47ZhYBuTCobDqX) — address that receives the staked KSMs on Kusama.
+
+Ethereum:
+
+[aKSMb](https://etherscan.io/address/0x84DA8e731172827fCB233B911678E2a82E27Baf2) — contract on the Ethereum network that mints or burns aKSMb tokens for the user in 1:1 ratio with the staked amount.
+
+[PolkadotPool](https://etherscan.io/address/0x59f767EC659E9FE01ebCf930465E2aD4Cc0F208e) — contract on the Ethereum network where the user sends their request to mint or burn aKSMb.
+
+## Staking Flow
+
+The flow consists of two parts: staking KSM and claiming aKSMb.
+
+### Staking KSM
+
+The workflow is both user- and ANKR-driven. The user part is Step 1 through 3, while the ANKR part is Step 3 through 5. User interacts with ANKR and Kusama via the ANKR frontend.
+
+Due to Polkadot backend architecture, we currently do not disclose our API endpoints to avoid scams or unregulated heavy traffic to the ANKR backend.
+
+1. User requests the ANKR deposit address via the ANKR backend.
+
+2. User sends KSMs to the obtained ANKR deposit address via the ANKR backend.
+
+3. User sends their extrinsic id to the ANKR backend.
+
+4. Once the transaction has been finalized, the user can see their KSM balance on the StakeFi page the stake was made from.
+
+5. ANKR backend monitor deposits to the ANKR deposit address. Once new KSMs are available for staking, the backend:
+   1. Transfer the KSMs to one of our treasury accounts if necessary.
```

Comment:
remove mention of treasure accounts. Suggest changing this indented part to bullets

Reply:
thx, replaced with Transfers the KSMs to one of ANKR dedicated accounts if necessary.